### ISSUE TITLE
Latest NoScript (5.1.5) causes issues

### ISSUE THREAD

**Issue by Noexpert** [Synzvato/decentraleyes#232]
With both noscript and decentraleyes enabled, one has to enable "file://" for decentraleyes to load the resource. Otherwise it is blocked.

**Comment by stachowskim** [comment]
So just enable "file://" in NoScript and it both will work correctly again?

**Comment by Synzvato** [comment]
Firefox Quantum users are advised to upgrade to NoScript ```v10.1.1``` (or higher). Firefox 52 ESR users, can keep using Decentraleyes ```v1.3.10```. It should perform well, and has no known vulnerabilities.

> So just enable ```file://``` in NoScript and it both will work correctly again?

@markstachowski Many thanks for weighing in. This could be a helpful temporary solution for Firefox 56 users, who are planning to upgrade to Firefox Quantum within the (relatively) near future.

**Comment by markstachowski** [comment]
Great, did noscript 10.x fix the issue or we still have to add file://

**Comment by Synzvato** [comment]
Yes, NoScript ```v10.1.1``` is essentially a complete rewrite. Upgrading should fix the ```file://``` issue.

**Comment by Noexpert** [comment]
Closing for now until I've had more time to check, but yes, it should be addressed with the new version.
